apps/mull-ui/src/app/pages/login/login.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { CustomTextInput } from '@mull/ui-lib';
+import * as Yup from 'yup';
+import { useFormik } from 'formik';
+import logo from '../../../assets/mull-logo.png';
+import { ReactComponent as TwitterIcon } from '../../../assets/icons/login-icons/twitter.svg';
+import { ReactComponent as GoogleIcon } from '../../../assets/icons/login-icons/google.svg';
+import { ReactComponent as FacebookIcon } from '../../../assets/icons/login-icons/facebook.svg';
+
+import './login.scss';
+
+export interface LoginProps {
+  history: History;
+}
+
+export const Login = ({ history }) => {
```

Comment:
No explicit export needed here. There's already an export default at the bottom of the file.

Reply:
Missing prop declaration.

Reply:
It seems that components generated with Nx have this double export by default, although possibly redundant, I don't think it's worth changing.

Reply:
I think it's the export option in the VS Code extension. Just unchecking it when generating the component should be fine.